lib/include/environment/freertos/rpmsg_env_specific.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 NXP
+ * All rights reserved.
+ *
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/**************************************************************************
+ * FILE NAME
+ *
+ *       rpmsg_env_specific.h
+ *
+ * DESCRIPTION
+ *
+ *       This file contains FreeRTOS specific constructions.
+ *
+ **************************************************************************/
+#ifndef RPMSG_ENV_SPECIFIC_H_
+#define RPMSG_ENV_SPECIFIC_H_
+
+#if defined(RL_USE_STATIC_API) && (RL_USE_STATIC_API == 1)
+#if (configSUPPORT_STATIC_ALLOCATION == 0)
+#warning You have configured RPMsg_Lite to use static API but FreeRTOS is not configured for static allocations! Please switch the configSUPPORT_STATIC_ALLOCATION to 1 in your FreeRTOSConfig.h file.
+#endif
+
+#include "FreeRTOS.h"
+
+typedef StaticSemaphore_t LOCK_STATIC_CONTEXT;
+typedef StaticQueue_t rpmsg_static_queue_ctxt;
+
+/* Queue object static storage size in bytes, should be defined as (RL_BUFFER_COUNT*sizeof(rpmsg_queue_rx_cb_data_t)) */
```

Comment:
I am concered about usage of rpmsg_queue_rx_cb_data_t without including header file with declaration.

Reply:
How about to move the rpmsg_queue_rx_cb_data_t typedef from the rpmsg_queue.h directly into the rpmsg_env_specific.h files?

Reply:
I agree

Reply:
Will you that? Thanks.

Reply:
Done